target/linux/ramips/image/mt7621.mk

```diff
@@ -2109,6 +2109,16 @@ define Device/rostelecom_rt-sf-1
 endef
 TARGET_DEVICES += rostelecom_rt-sf-1
 
+define Device/ruijie_rg-ew1200g-pro
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  IMAGE_SIZE := 20480k
```

Comment:
Where did this value come from? ```20480k``` (0x1400000) is larger than the size of the flash chip on this device.